datajoint/admin.py

```diff
@@ -39,6 +39,10 @@ def kill(restriction=None, connection=None, order_by=None):  # pragma: no cover
     if connection is None:
         connection = conn()
 
+    if order_by is not None and not isinstance(order_by, str):
+        # partial order_by parsing - we don't do key manip, etc. like fetch
```

Comment:
This comment is unnecessary (no pertinent information). There is no parsing as such. You could say "compare with order_by in fetch" instead

Reply:
it's pertinent in that the order by functions differently and this explains why it was done this way - not sure of a better place to document this kind of trivia and the alternative is omission. if omission is preferred, wil proceed.

Reply:
ok, moved to single line and minimalized

Reply:
What does "partial order_by parsing" mean? What's missing?

Reply:
What's "key manip"?

Reply:
clearly, need to 'cf. Fetch.fetch()` :) 

fetch.fetch does:

```
# expand "KEY" or "KEY DESC"
order_by = list(_flatten_attribute_list(self._expression.primary_key, order_by))
```

which doesn't really apply so much here but technically could

since processlist isn't a datajoint table, there is no primary key, and so using/implementing this logic & associated 'KEY' expansion that happens in _flatten_attribute_list would be superflouous.

however, since direction was to 'make it like fetch', as implemented, this update fails the spec.

Reply:
But the reader of this code does not know this. This logic can be understood simply without knowing anything about fetch.  The comment wording makes it sound like one needs to understand fetch in order to understand this code.

Reply:
This code does not do "key manip", so there is no need to bring it up.

Reply:
fair point, but the reader should also understand 'how order_by works in datajoint', and 'why this code deviates' which comprises 2 mechanisms: dj.kill shorthand and also 'real' fetch.fetch.

anyway, its amended.

Reply:
partially this was a defensive comment against the potential 'but this doesn't actually work like fetch' statement. this discussion replaces the need for the comment to transmit that understanding at least for the current devteam.

Reply:
This `order_by` does not have to do much with `fetch` and it's not helpful to get the reader to understand fetch for understanding `kill`. `kill` does not have anything to do with queries and fetch logic. I would prefer to comment here -- it only distracts.

Reply:
makes sense - which (in retrospect) is why i initially had left out the multiple order by option to begin with. question here was to what extent does it need to be 'like real fetch' since initial partial implementation was deemed 'too partial' in it's omission.

so - should I zap the updated short form comment as well?

Reply:
Thanks. yes, these `admin` utilities are rather general and have little to do with the rest of datajoint, so there is no need explain datajoint.